src/app/shared/components/time-bar/time-bar.component.scss

```diff
@@ -0,0 +1,5 @@
+@import 'src/_themes.scss';
```

Comment:
La languette pour les settings est sur le composant en mobile

Reply:
I've test it on mobile device, it displays as expected.
![mobile_disp](https://user-images.githubusercontent.com/40590760/78046856-4192dd80-73aa-11ea-9e11-688749bd1b2b.PNG)

Reply:
The component should be displayed under the green settings gear.